scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -408,16 +408,27 @@ def add_configurations(self):
                             Path(ZEPHYR_BASE) / 'subsys' / 'testsuite'],
                             board_roots=board_roots, board=None, board_dir=None)
         v1_boards = list_boards.find_boards(lb_args)
-        v2_boards = list_boards.find_v2_boards(lb_args)
```

Comment:
so the find_v2_boards are not used any more? can we remove this function totally?

Reply:
Not used in Twister itself, but still present in some scripts outside of it, so we cannot remove it.

https://github.com/zephyrproject-rtos/zephyr/blob/45205f865e18decbb9568c4691686dc679cbe961/scripts/west_commands/boards.py#L93-L97
https://github.com/zephyrproject-rtos/zephyr/blob/45205f865e18decbb9568c4691686dc679cbe961/scripts/ci/check_compliance.py#L426-L440